content/app_mesh/gettingstarted/prerequistes.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "Pre-requistes Config"
+description = "Pre-requiste steps for setting up and running workloads in App Mesh"
+weight = 2
++++
+
+Before we get started lets create basic configurations for our application
+
+ECS Service Roles
+```bash
+aws iam get-role --role-name "AWSServiceRoleForECS" || aws iam create-service-linked-role --aws-service-name "ecs.amazonaws.com"
+```
+
+We will create our own repositories in ECR, so we can avoid  Docker Hub requests limits
+
+```bash
+
+# echo '======frontend====='
```

Comment:
we can remove docker pull/push steps because we have the images stored upstream in ecr public now.

Reply:
> we can remove docker pull/push steps because we have the images stored upstream in ecr public now.

Actually this will remove some steps in different parts of the workshop.

Reply:
Changed